content/2021/2021-05/2021-05-24-pulsar-weekly.md

```diff
@@ -0,0 +1,150 @@
+
+- [Java][Schema] Enable disabled tests about `AutoConsume` and `KeyValue` and add the test about null values.
+
+    https://github.com/apache/pulsar/pull/10626 ([eolivelli](https://api.github.com/users/eolivelli))
+
+- [Java][Schema] Add more tests about `KeyValue` and NULL values.
+
+    https://github.com/apache/pulsar/pull/10609 ([eolivelli](https://api.github.com/users/eolivelli))
+
+- [Broker] Set up the Pulsar cluster with `MetadataStore`.
+
+    https://github.com/apache/pulsar/pull/10600 ([fantapsody](https://api.github.com/users/fantapsody))
+
+### Notable Bug Fix
+
+- [Admin] Fix the issue that when getting the partition metadata of a non-existent topic, it returns `0`.
+
+    https://github.com/apache/pulsar/pull/10601 ([BewareMyPower](https://api.github.com/users/BewareMyPower))
+
+- [Admin] Convert the bundle split operation into an HTTP asynchronous operation.
+
+    https://github.com/apache/pulsar/pull/10619 ([merlimat](https://api.github.com/users/merlimat))
+
+- [pulsar-client] Fix the issues in the pulsar-client shading configuration.
+
+    https://github.com/apache/pulsar/pull/10614 ([merlimat](https://api.github.com/users/merlimat))
+
+- [pulsar-client] Fix the issue that the `MessageCrypto` interface should not expose the Netty `ByteBuf` class in the API.
+
+    https://github.com/apache/pulsar/pull/10616 ([merlimat](https://api.github.com/users/merlimat))
+
+- [Build] Fix distribution build errors caused by the denied permission.
+
+    https://github.com/apache/pulsar/pull/10641 ([timmyyuan](https://api.github.com/users/timmyyuan))
+
+- [Test] Fix the failed `TopicPoliciesTest.testMaxSubscriptionsFailFast` test.
+
+    https://github.com/apache/pulsar/pull/10640 ([eolivelli](https://api.github.com/users/eolivelli))
+
+- [Broker] Fix the issue that consumer-related topic stats are only available when the consumer or the reader is connected.
+
+    https://github.com/apache/pulsar/pull/10644 ([dlg99](https://api.github.com/users/dlg99))
+
+- [Common] Fix the `ConcurrentOpenLongPairRangeSet` to remove data for all ranges.
+
+    https://github.com/apache/pulsar/pull/10656 ([315157973](https://api.github.com/users/315157973))
+
+### Event / News
+
+- Pulsar Virtual Summit North America 2021
+
+    - Sign-up: https://hopin.com/events/pulsar-summit-north-america-2021
+
+- [Pulsar Office Hour] Monthly live stream about Pulsar best practices, use cases, and more.
+
+  - 05/19: https://www.youtube.com/watch?v=vGg9wiYjvUM
+
+- [TGIP] Weekly live stream about Pulsar and its ecosystem.
+
+  - All video recordings are available at [here](https://streamnative.io/resource#tgip).
+
+- [Intro to Apache Pulsar 101] Monthly live stream about Pulsar, including concepts, architecture, and design.
+
+    - All video recordings are available at [here](https://streamnative.io/en/resource#intro-to-apache-pulsar-101).
+
+### Blog / Article
+
+- What’s New in Apache Pulsar 2.7.2
```

Comment:
Do you collect info between "2021-05-17 ~ 2021-05-23"?

Reply:
I think it's better to cover this blog in this weekly report since the release manager has announced the release of 2.7.2. If you do not agree, i can remove this section, as there is no new blog available last week.

Reply:
I think it's ok to add it here.